README.md

```diff
@@ -168,17 +159,7 @@ If you like this project...
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
 
 I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly fill out these sections as the previous reviewers requested. It would make your documentation more professional